lsp-mode.el

```diff
@@ -7905,6 +7907,26 @@ in place."
     (error "Unable to find `gzip' on the path, please either customize `lsp-gunzip-script' or manually decompress %s" gz-file))
   (shell-command (format lsp-gunzip-script gz-file)))
 
+;; tar.gz decompression
+
+(defconst lsp-ext-tar-script "bash -c 'mkdir -p %2$s; tar xf %1$s --directory=%2$s'"
+  "Script to decompress a .tar.gz file.")
+
+(defcustom lsp-tar-script (cond ((executable-find "tar") lsp-ext-tar-script)
```

Comment:
I missed that in the original PR. We should not run executable-find on the top level. We could use a function and `lsp-resolve-value` to make the execution lazy.